app/src/main/res/values-ru/strings.xml

```diff
@@ -22,77 +22,77 @@
-    <string name="after">after</string>
-    <string name="before">before</string>
-    <string name="morse_code">morse code</string>
+    <string name="choose_practice_file">выбрать тренировочный файл</string>
+    <string name="trainingStartText">Запустить</string>
+    <string name="trainingStopText">Остановить</string>
+    <string name="repeat">Повторять по</string>
+    <string name="times">раз(-а)</string>
+    <string name="audio_hint">Произносить ответ</string>
+    <string name="after">до</string>
+    <string name="before">после</string>
+    <string name="morse_code">сообщения азбукой Морзе</string>
 
     <!-- Strings related to about activity -->
-    <string name="about_version_label">Version</string>
+    <string name="about_version_label">Версия</string>
     <string name="about_version_dummy_value">0.00</string>
-    <string name="app_description">A little program to assist me in learning morse code.</string>
-    <string name="help_create_word_list">To create your own word list create a new text file with the \".txt\" suffix in the \"Claus\' Morse Trainer\" directory of the internal storage. Each line should be a character, word or phrase that you want to learn. If the spoken words differ from the morse text then separate the morse text with a vertical pipe \"|\" from the spoken text (e.g. \"fb|fine business\").</string>
-    <string name="support">To report issues, get support or view the source code please visit the following link.</string>
+    <string name="app_description">Небольшая программа которая помогает мне при изучении азбуки Морзе.</string>
+    <string name="help_create_word_list">Чтобы составить свой собственный список слов, создайте новый текстовый файл с расширением \".txt\" в папке \"Claus\' Morse Trainer\" на Внутреннем хранилище. Каждая строка файла должна представлять из себя символ, слово или фразу, которую вы хотите отработать. Если вы хотите, чтобы произносимый голосом ответ отличался от сообщения азбукой Морзе, отделите сообщение вертикальной чертой (\"|\") от произносимого (например: \"ДСВ|до свидания\").</string>
+    <string name="support">Чтобы сообщить о проблемах, получить помощь или посмотреть исходный код, пройдите по следующей ссылке:</string>
     <string name="support_page_url">https://github.com/cniesen/morsetrainer</string>
-    <string name="contributors_heading">Contributions</string>
-    <string name="contributors">&#8226; Nathan Crapo: improved dit &amp; dah sounds.</string>
-    <string name="note_of_thanks">Special thanks to my CW Academy advisor Dan and buddy Matt for not giving up on me.</string>
-    <string name="signature">Best wishes to your morse study,\n\tClaus Niesen (AE0S)</string>
+    <string name="contributors_heading">Вклад в проект внесли:</string>
+    <string name="contributors">&#8226; Натан Крапо (Nathan Crapo): улучшил звучание точки и тире.\n&#8226; Igorello74 (R8AABM): способствовал поддержке русских символов, перевёл на русский</string>
```

Comment:
Don't you mind if I humbly note myself? I'm only saying that I had assisted the support of Russian and had made a Russian translation.